readthedocs_theme/templates/includes/topnav.html

```diff
@@ -1,10 +1,55 @@
-{% macro menu_main() %}
-  <a class="item{% if page and page.slug == 'features' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/features/">Features</a>
-  <a class="item{% if page and page.slug == 'pricing' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/pricing/">Pricing</a>
-  <a class="item{% if page and 'blog' in page.slug %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/blog/">Blog</a>
+{# The main product menu, for all high priority information #}
+{% macro menu_product(with_more=False) %}
+  <div class="header">About</div>
+  <a class="item" href="{{ SITEURL }}/pricing/">
+    <i class="fad fa-diagram-venn primary icon"></i>
+    Pricing
+  </a>
```

Comment:
I might pull this out into it's own top-level, since I think it's the most important page?

Reply:
Yeah was debating this, but think either spot works too. I'll give a quick try, but if the menu is too wide, maybe I'll merge this for now and see if pricing pageviews drops.